opus/application/static_media/js/opus.js

```diff
@@ -646,13 +659,17 @@ var opus = {
         $(document).on("keydown click", function(e) {
             if ((e.which || e.keyCode) == 27) {
                 // ESC key - close modals and help panel
-                $(".op-confirm-modal").modal('hide');
+                // But don't close "#op-browser-version-msg" and "#op-browser-size-msg"
+                if (!$(".op-confirm-modal").is("#op-browser-version-msg") &&
```

Comment:
i wonder if it would be useful in the future to have a class or data element on the modal that identifies it as "close-able on esc", instead of having to check the id of the modal.    Then you could just do something like:
if ($(".op-confirm-modal").data("action") === "esc")    more or less, doesn't have to be exactly this.